src/Dropdown/index.jsx

```diff
@@ -138,14 +137,17 @@ class Dropdown extends React.Component {
           type="button"
           inputRef={(toggleElem) => { this.toggleElem = toggleElem; }}
         />
-        <ul
+        <div
```

Comment:
Although the convention is to use a `ul` for dropdown lists, @cptvitamin says it's just fine not to.